api/config.js

```diff
@@ -63,6 +63,7 @@ config.auth = {};
 config.database = {};
 config.database.primary = { pool: {} };
 config.mail = {};
+config.tokenExpiration = {};
```

Comment:
Can we change this to `config.token = { expiration: {} }` and put the expirations in there? `config.tokenExpiration` seems uncomfortably specific for a top-level config object